standalone/RL_plus_script.py

```diff
@@ -103,14 +116,20 @@ def action(self, action):
         return self.actions[action]
 
 
-def train():
-    env = gym.make(TRAIN_ENV)
-    env = PovOnlyObservation(env)
-    env = ActionShaping(env, always_attack=True)
+def train(wandb_project_name=None):
+    if wandb_project_name:
+        import wandb
+        wandb.init(
+            project=wandb_project_name,
+            sync_tensorboard=True,
+            name=experiment_name,
+            monitor_gym=True,
+            save_code=True,
+        )
 
+    env = DummyVecEnv([make_env(i) for i in range(1)])
     # For all the PPO hyperparameters you could tune see this:
-    # https://github.com/DLR-RM/stable-baselines3/blob/6f822b9ed7d6e8f57e5a58059923a5b24e8db283/stable_baselines3/ppo/ppo.py#L16
```

Comment:
why delete this line?

Reply:
Added back. For some reasons I thought I added the line